src/Register.js

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import {validateAll} from 'indicative/validator';
+
+export class Register extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            login: '',
+            email: '',
+            password: '',
+            password_confirmation: '',
+            errors: {}
+        };
+    }
+
+    onClick = () => {
+
+        const messages = {
```

Comment:
messages and rules look like constant values and might be moved out of the component.